force-app/main/default/classes/EncryptionRecipes.cls

```diff
@@ -0,0 +1,436 @@
+     * In this case the initialization vector is specified by the sender. It needs to be random and 16 bytes (128 bits).
+     * @param dataToEncrypt Blob that contains the data to encrypt
+     * @return Blob
+     * @example
+     * Blob initializationVector = EncryptionRecipes.generateInitializationVector();
+     * Blob dataToEncrypt = Blob.valueOf('Test data');
+     * Blob encryptedData = EncryptionRecipes.encryptAES256Recipe(dataToEncrypt, initializationVector);
+     * System.debug(EncodingUtil.base64Encode(encryptedData));
+     **/
+    @AuraEnabled
+    public static Blob encryptAES256Recipe(
+        Blob dataToEncrypt,
+        Blob initializationVector
+    ) {
+        // Call Crypto.encryptWithManagedIV specifying the selected AES Algorithm
+        Blob encryptedData = Crypto.encrypt(
+            EncryptionRecipes.AESAlgorithm.AES256.name(),
+            AES_KEY,
+            initializationVector,
+            dataToEncrypt
+        );
+        // Combine the encryptedData and initializationVector to send both values to the receiver
+        String blobsAsHex =
+            EncodingUtil.convertToHex(initializationVector) +
+            EncodingUtil.convertToHex(encryptedData);
+        return EncodingUtil.convertFromHex(blobsAsHex);
+    }
+
+    /**
+     * @description Encrypts data using AES algorithm, which needs a symmetric key to be shared with the receiver.
+     * In this case the sender needs to share the initialization vector with the receiver.
+     * @param dataToDecrypt Blob that contains the data to be decrypted
+     * @return Blob
+     * @example
+     * Blob decryptedData = EncryptionRecipes.decryptAES256Recipe(encryptedData);
+     * System.debug(decryptedData.toString());
+     **/
+    @AuraEnabled
+    public static Blob decryptAES256Recipe(Blob dataToDecrypt) {
+        // Convert received blobs to hex
+        String blobsAsHex = EncodingUtil.convertToHex(dataToDecrypt);
+        // Substract first two bytes (initialization vector)
+        String initializationVectorString = blobsAsHex.substring(0, 32);
+        // Substract the rest (encrypted data)
+        String encryptedDataString = blobsAsHex.substring(32);
+        // Convert both hex values to Blob
+        Blob initializationVector = EncodingUtil.convertFromHex(
+            initializationVectorString
+        );
+        Blob encryptedData = EncodingUtil.convertFromHex(encryptedDataString);
+        // Call Crypto.decryptWithManagedIV specifying the selected AES Algorithm
+        return Crypto.decrypt(
+            EncryptionRecipes.AESAlgorithm.AES256.name(),
+            AES_KEY,
+            initializationVector,
+            encryptedData
+        );
+    }
+
+    /**
+     * @description Aux method to generate a random initialization vector.
+     * @return Blob
+     **/
+    public static Blob generateInitializationVector() {
+        // Generating IV this way until there's an Apex method for it
```

Comment:
Do we need to explicitly say do not use the same value for the IV and AES key? There is a very, very small chance that they end up the same by random chance, but I don't believe any implementations actually check for that.

I think the additional of this method makes it clear that the IV and AES key are not related, but want your input since you work more closely with developers.